garage/tf/algos/npo.py

```diff
@@ -300,6 +301,10 @@ def _build_policy_loss(self, i):
                         policy_dist_info,
                         name="lr")
 
+                    ll = pol_dist.log_likelihood_sym(
+                        i.action_var, policy_dist_info, name="log_likelihood")
+
+                    vanilla = ll * advantages * i.valid_var
```

Comment:
i think it might be more clear if you moved these inside the `if` statement, even if it means copying a couple lines